bcmc/src/main/java/com/adyen/checkout/bcmc/internal/ui/model/BcmcComponentParamsMapper.kt

```diff
@@ -40,12 +48,23 @@ internal class BcmcComponentParamsMapper(
             isHolderNameRequired = isHolderNameRequired ?: false,
             shopperReference = shopperReference,
             isStorePaymentFieldVisible = isStorePaymentFieldVisible ?: false,
+            addressParams = AddressParams.None,
+            installmentParams = null,
+            kcpAuthVisibility = KCPAuthVisibility.HIDE,
+            socialSecurityNumberVisibility = SocialSecurityNumberVisibility.HIDE,
+            cvcVisibility = cvcVisibility ?: CVCVisibility.ALWAYS_HIDE,
+            storedCVCVisibility = StoredCVCVisibility.HIDE,
+            supportedCardBrands = listOf(
```

Comment:
Are these the only brands BCMC supports? And how could we future proof this? Maybe BCMC adds more brands later

Reply:
Changed. We're using `paymentMethod` and if it fails we fall back to the hardcoded list.